view/src/components/produce.js

```diff
@@ -230,10 +220,112 @@ class produce extends Component {
 		});
 	}
 
+	// This function updates the string that the produce array will be searched for.
+	// This string value can contain any produce field (name, weight, interrnal shipment numbers) and will still return
+	// 	the appropriate filtered results.
+
 	handleSearch = event => {
-		const { value } = event.target;
-		this.setState({ value });
-	  };
+    const { value } = event.target;
+    this.setState({ value });
+	};
+
+	// Handles the rendering of filtered produce results into React cards
+	handleResultsRender = (results) => {
+		const { classes } = this.props;
+		return (
+			<div>
+			{results.map((produce) => (
+			<Grid item xs={12} key={produce.produceId}>
+				<Card className={classes.root} variant="outlined">
+				<CardContent>
+					<Typography variant="h5" component="h2">
+					{produce.name}
+					</Typography>
+					<Box
+					display="flex"
+					flexDirection="row"
+					flexWrap="wrap"
+					p={0}
+					m={0}
```

Comment:
This is my biggest gripe with React/MUI. p and m? What kind of names are these?? I'm guessing `p` is padding? `m` is...margin? (If they are both 0 maybe you can omit this?)

I don't think I'm asking for anything to change, just expressing that this is not readable/scalable CSS, I'm really not thrilled with their styling system.

Reply:
https://material-ui.com/system/spacing/ noted, padding and margin can be used instead of the shorthand. I went ahead and replaced it for this file, and can do the same when i add search to the other pages. looping in Andrew if he wants to add this in any other UI areas - @andrewhojel